packages/clay-navigation-bar/src/index.tsx

```diff
@@ -118,6 +122,7 @@ function ClayNavigationBar({
 						<span className="navbar-text-truncate">
 							{triggerLabel}
 						</span>
+						<span className="sr-only">{itemLabel}</span>
```

Comment:
Instead of `sr-only`, I think it's better if we use `aria-label` on the button instead. I think we need to be able to pass in a custom message to the `aria-label` for translation purposes.

Reply:
Ah I see, I will make that change

Reply:
@pat270 I made the change but for some reason I was not able to run yarn develop to test it, are you able to test this one? I am not sure why the CI/test check is failing as well, it looks unrelated to my changes

Reply:
@SelenaAungst you need to recreate the test snapshots by running `yarn test -u`. Since you updated the HTML this is reflected in the snapshot tests which are a way to ensure that the DOM remains the same. We only update the snapshot when the DOM change is intentional like your use case, just to avoid regressions.

In relation to this we should move this to be configured via props in the component, something like:
```jsx
<ClayNavigationBar
  messages={{
    close: 'Close',
    open: 'Open',
    trigger: '{0} Menu, Current Page: {1}',
  }}
>
 {...}
</ClayNavigationBar>
```
So internally you can use these properties to form the `aria-label` of the trigger.
```jsx
<ClayButton
  aria-expanded={expanded}
  aria-label={sub(
    messages.trigger,
    [expanded ? messages.close : messages.open, triggerLabel]
  )}
```

Reply:
Ah got it, thanks @matuzalemsteles! So do the ClayNavigationBar props still go in index.tsx or somewhere else? I will make the change to aria-label and run yarn test -u after this is complete.

Reply:
> So do the ClayNavigationBar props still go in index.tsx or somewhere else?

Yes, you just need to define this new API type in the component props as well and add the type.

Reply:
Thanks @matuzalemsteles! I tried to add these changes and update tests. I am not positive this is correct so please let me know if there are any additional changes I can make.